### ISSUE TITLE
Rebooting homebridge makes Solis created automation disappear from Homekit

### ISSUE THREAD

**Issue by Bluey05** [szlaskidaniel/homebridge-solis5g-battery#2]
Version plug-in installed: v1.1.10
Homebridge version: v1.8.1
Homebridge UI: v4.56.2
Nodejs: 20.12.2

I like this plug-in a lot and use it for some daily automations based on the battery charge derived from this plug-in.
The down side is I need to re-add the triggers for automations based on battery "speed" with the SOLIS plug-in created in Homekit as they disappear when Homebridge is rebooted.
In Short, the triggers using 'battery "speed" disappear and the automation is then disabled.
